docs/specs/helpers.md

```diff
@@ -161,7 +161,7 @@ The operations need to be chained using right multiplication instead of left mul
 
 Slots are a way to define a property value once and use the value in multiple
 properties. Slot definitions are in a dictionary, the slot definition key is the
-key that is used to match all properties with an `sid` field to the same key for
+key that is used to match all properties with a `sid` field to the same key for
```

Comment:
I think both these are valid depending on whether you say each letter individually or the whole thing as a single syllable 🤣 .

Reply:
we should use `ar` in this case because `r` looks like half an `n` :P